Compile: UCT error in rc_mlx5_devx.c (Fedora 32, GCC 10)

### Describe the bug

Compile (warning as) error in rc_mlx5_devx.c.

### Steps to Reproduce
- `make`
- UCX version used (from github branch XX or release YY) + UCX configure flags (can be checked by `ucx_info -v`)

github master @ 2b51ec57589994f482e9c8caa15cd1ce637ea76b

- **Any UCX environment variables used**

### Setup and versions
- OS version (e.g Linux distro) + CPU architecture (x86_64/aarch64/ppc64le/...)
   - `cat /etc/issue` or `cat /etc/redhat-release` + `uname -a`

```
Fedora release 32 (Thirty Two)
```
```
Linux fedvm 5.6.12-300.fc32.x86_64 #1 SMP Mon May 11 16:47:13 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

```
make[4]: Entering directory '/home/tc/src/ucx/build/src/uct/ib'
  CC       rc/accel/libuct_ib_la-rc_mlx5_devx.lo
../../../../ucx-git/src/uct/ib/rc/accel/rc_mlx5_devx.c: In function ‘uct_rc_mlx5_devx_iface_init_events’:
../../../../ucx-git/src/uct/ib/rc/accel/rc_mlx5_devx.c:107:42: error: passing argument 4 of ‘ucs_async_set_event_handler’ from incompatible pointer type [-Werror=incompatible-pointer-types]
  107 |                                          uct_rc_mlx5_devx_iface_event_handler, iface,
      |                                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                          |
      |                                          void (*)(int,  void *)
In file included from /home/tc/src/ucx/build/../ucx-git/src/uct/api/uct.h:17,
                 from /home/tc/src/ucx/build/../ucx-git/src/uct/ib/base/ib_device.h:12,
                 from ../../../../ucx-git/src/uct/ib/rc/accel/rc_mlx5_common.h:10,
                 from ../../../../ucx-git/src/uct/ib/rc/accel/rc_mlx5.h:10,
                 from ../../../../ucx-git/src/uct/ib/rc/accel/rc_mlx5.inl:7,
                 from ../../../../ucx-git/src/uct/ib/rc/accel/rc_mlx5_devx.c:11:
/home/tc/src/ucx/build/../ucx-git/src/ucs/async/async_fwd.h:51:75: note: expected ‘ucs_async_event_cb_t’ {aka ‘void (*)(int,  int,  void *)’} but argument is of type ‘void (*)(int,  void *)’
   51 |                                    int events, ucs_async_event_cb_t cb,
      |                                                ~~~~~~~~~~~~~~~~~~~~~^~

cc1: all warnings being treated as errors
```

--- a/src/uct/ib/rc/accel/rc_mlx5_devx.c
+++ b/src/uct/ib/rc/accel/rc_mlx5_devx.c
@@ -45,7 +45,7 @@ ucs_status_t uct_rc_mlx5_devx_iface_subscribe_event(uct_rc_mlx5_iface_common_t *
 }
 
 #if HAVE_DECL_MLX5DV_DEVX_SUBSCRIBE_DEVX_EVENT
-static void uct_rc_mlx5_devx_iface_event_handler(int fd, void *arg)
+static void uct_rc_mlx5_devx_iface_event_handler(int fd, int events, void *arg)
 {
     uct_rc_mlx5_iface_common_t *iface = arg;
     uct_ib_md_t *md                   = uct_ib_iface_md(&iface->super.super);

